kordac/processors/errors/TagNotMatchedError.py

```diff
@@ -0,0 +1,15 @@
+from kordac.processors.errors import Error
+
+class TagNotMatchedError(Error):
```

Comment:
This is a little big vague, you are referring to not finding an end tag right? Can you change this to a more explicit class name?